testrunner/js/helpers.js

```diff
@@ -28,4 +28,40 @@
 function isFunction(functionToCheck) {
 	var getType = {};
 	return functionToCheck && getType.toString.call(functionToCheck) === '[object Function]';
+}
+
+/**
+ * Counts the number of keys in an object.
+*/
+function numberOfKeys(obj) {
```

Comment:
Nice, that's good to know. I think I'll keep it the way it is for now, but I'll definitely read up on how Object.keys works, and if it's faster I'll swap it out.